crates/rust-analyzer/src/config.rs

```diff
@@ -61,233 +61,312 @@ pub enum MaxSubstitutionLength {
     Limit(usize),
 }
 
-// Defines the server-side configuration of the rust-analyzer. We generate
-// *parts* of VS Code's `package.json` config from this. Run `cargo test` to
-// re-generate that file.
+// Defines the server-side configuration of the rust-analyzer. We generate *parts* of VS Code's
+// `package.json` config from this. Run `cargo test` to re-generate that file.
```

Comment:
In general, Ive wrapped these at 100 chars as the changes were otherwise difficult to read at times with various wrap points across each item. I've also put a single blank line between each item instead of 0-3, this makes it easier to read as the comment is more obviously grouped with the following item rather than the previous.

I also removed the vertically aligned defaults (` = true` etc.), as this formatting doesn't appear elsewhere in the source code and doesn't make the values easier to read.